manifests/role/hathitrust/dev/app_host_and_ruby.pp

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2020 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# Grog, buzz
```

Comment:
Giving buzz `nebula::role::hathitrust::dev::app_host_and_ruby` instead of `nebula::role::hathitrust::dev` will bring in both the `ruby` profile and the `named_instances` profile. Not sure whether adding the named_instances profile will cause problems.